.github/workflows/test-jinad.yml

```diff
@@ -2,10 +2,8 @@ name: Test jinad
 
 on:
   pull_request:
-    #branches:
-    #  - master
-    branches-ignore:
-      - '**'
+    branches:
```

Comment:
Is this related to the problem we were having with jinad tests failing? Are they ok now?

Reply:
they are almost ready